### ISSUE TITLE
xfdashboard is extremely slow on a notebook

### ISSUE THREAD

**Issue by GreenLunar** [gmc-holle/xfdashboard#96]
xfdashboard-0.3.9
https://wiki.debian.org/InstallingDebianOn/Dell/Inspiron1010-Mini10#lspci

```
$ lspci
00:00.0 Host bridge: Intel Corporation System Controller Hub (SCH Poulsbo) (rev 07)
00:02.0 VGA compatible controller: Intel Corporation System Controller Hub (SCH Poulsbo) Graphics Controller (rev 07)
00:1b.0 Audio device: Intel Corporation System Controller Hub (SCH Poulsbo) HD Audio Controller (rev 07)
00:1c.0 PCI bridge: Intel Corporation System Controller Hub (SCH Poulsbo) PCI Express Port 1 (rev 07)
00:1c.1 PCI bridge: Intel Corporation System Controller Hub (SCH Poulsbo) PCI Express Port 2 (rev 07)
00:1d.0 USB controller: Intel Corporation System Controller Hub (SCH Poulsbo) USB UHCI #1 (rev 07)
00:1d.1 USB controller: Intel Corporation System Controller Hub (SCH Poulsbo) USB UHCI #2 (rev 07)
00:1d.2 USB controller: Intel Corporation System Controller Hub (SCH Poulsbo) USB UHCI #3 (rev 07)
00:1d.7 USB controller: Intel Corporation System Controller Hub (SCH Poulsbo) USB EHCI #1 (rev 07)
00:1f.0 ISA bridge: Intel Corporation System Controller Hub (SCH Poulsbo) LPC Bridge (rev 07)
00:1f.1 IDE interface: Intel Corporation System Controller Hub (SCH Poulsbo) IDE Controller (rev 07)
02:00.0 Ethernet controller: Realtek Semiconductor Co., Ltd. RTL8101E/RTL8102E PCI Express Fast Ethernet controller (rev 02)
03:00.0 Network controller: Broadcom Corporation BCM4312 802.11b/g LP-PHY (rev 01)
```

```
VGA compatible controller
/0/100/2


product: System Controller Hub (SCH Poulsbo) Graphics Controller [8086:8108]
vendor: Intel Corporation [8086]
bus info: pci@0000:00:02.0
version: 07
width: 32 bits
clock: 33MHz
capabilities:
	Power Management,
	Message Signalled Interrupts,
	vga_controller,
	bus mastering,
	PCI capabilities listing,
	extension ROM
configuration:
	driver: gma500
	latency: 0
resources:
	irq: 16
	memory: d8100000-d817ffff
	ioport: 1800(size=8)
	memory: d0000000-d7ffffff
	memory: d8380000-d839ffff
```

**Comment by gmc-holle** [comment]
As far as I could read on other websites Intel's GMA500 GPU seems not to be one of the faster ones. At ArchLinux Wiki (https://wiki.archlinux.org/index.php/Poulsbo) I also read that 3D is not possible. If this is true then I believe the OpenGL capabilities of your desktop will be software rendered and therefore slow .... really slow.

Even with all options to speed up xfdashboard (delayed search start, low priority of live window content creation, reducing unneccessary texture loadings etc.) I don't think it will get better but I may be wrong.

I see that you refer to an older development release (0.3.9) of xfdashboard. If you like to stay on the development release please consider to upgrade to latest git to get the latest changes like "low priority of window content creation". Or update to 0.5.0 and start xfdashboard with the environment variable XFDASHBOARD_WINDOW_CONTENT_RESUME_ON_IDLE set to 1

```
XFDASHBOARD_WINDOW_CONTENT_RESUME_ON_IDLE=1 xfdashboard
```

Add `-d` to the command above to start xfdashboard in daemon mode what I would prefer at slower machines.

**Comment by GreenLunar** [comment]
Unfortunately, the required dependencies for 0.5.0 are not available in Slackware 14.1.

I will let you know if it works better.
